audio/nrsc5/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        theori-io nrsc5 f1a32a551471dc5045193027e4c828746c5790fd
+
+revision            0
+
+categories          audio
+maintainers         {netjibbing.com:blake @trodemaster}
+license             GPL
```

Comment:
`GPL` means any version of the GPL can be used. If this is not the case, specify the allowed version range.

Reply:
Updated to GPL-3+